StandaloneMmPkg/Library/StandaloneMmMemLib/StandaloneMmMemLib.h

```diff
@@ -15,6 +15,14 @@
 #include <Library/BaseMemoryLib.h>
 #include <Library/DebugLib.h>
 
+typedef struct {
+  EFI_PHYSICAL_ADDRESS    Base;
+  UINT64                  Length;
+} NON_MM_MEMORY_MAP;

```

Comment:
NON_MM_MEMORY_MAP is only used by X86.c, right?
Then can you move the structure to X86.c?

Reply:
Moved the structure to X86.c